### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Margen67**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,12 @@ This will show an xex address when a breakpoint is hit, although there is curren
 
 ### Setting up Cheat Engine
 Memory Breakpoints can be set in Cheat Engine or MSVC with `emit_source_annotations`.
-  * [Camera searching with Cheat Engine](https://github.com/emoose/xenia/issues/9)
+  * Cheat Engine now includes Big Endian types (Must Enable Them).
```
Comment (BillieBlueberry) [inline README.md:101]: Cheat Engine now includes Big Endian types (must enable them).

File: README.md [hunk 2]
```diff
@@ -98,7 +98,12 @@ This will show an xex address when a breakpoint is hit, although there is curren
...
+You can enable them by going to Edit > Options > Extra Custom Types
```
Comment (BillieBlueberry) [inline README.md:102]:
```suggestion
  * To search the games memory change start in the 'Memory Scan Options' panel to 0000000100000000, and stop to 0000000200000000.
```

**Comment by oSerenity** [comment]
@illusion0001 suggest RCX

**Comment by illusion0001** [comment]
@oSerenity rcx doesn't have xex addr

**Review by oSerenity — APPROVED** [review]

**Pull request merged**